constants/quizPrice.ts

```diff
@@ -0,0 +1,42 @@
+export const priceToTemplates = [
+  { value: 200000, label: "200,000 $" },
+  { value: 195000, label: "195,000 $" },
+  { value: 190000, label: "190,000 $" },
+  { value: 185000, label: "185,000 $" },
+  { value: 180000, label: "180,000 $" },
+  { value: 175000, label: "175,000 $" },
+  { value: 170000, label: "170,000 $" },
+  { value: 165000, label: "165,000 $" },
+  { value: 160000, label: "160,000 $" },
+  { value: 155000, label: "155,000 $" },
+  { value: 150000, label: "150,000 $" },
+  { value: 145000, label: "145,000 $" },
+  { value: 140000, label: "140,000 $" },
+  { value: 135000, label: "135,000 $" },
+  { value: 130000, label: "130,000 $" },
+  { value: 125000, label: "125,000 $" },
+  { value: 120000, label: "120,000 $" },
+  { value: 115000, label: "115,000 $" },
+  { value: 110000, label: "110,000 $" },
+  { value: 105000, label: "105,000 $" },
+  { value: 100000, label: "100,000 $" },
+  { value: 95000, label: "95,000 $" },
+  { value: 90000, label: "90,000 $" },
+  { value: 85000, label: "85,000 $" },
+  { value: 80000, label: "80,000 $" },
+  { value: 75000, label: "75,000 $" },
+  { value: 70000, label: "70,000 $" },
+  { value: 65000, label: "65,000 $" },
+  { value: 60000, label: "60,000 $" },
+  { value: 55000, label: "55,000 $" },
+  { value: 50000, label: "50,000 $" },
+  { value: 45000, label: "45,000 $" },
+  { value: 40000, label: "40,000 $" },
+  { value: 35000, label: "35,000 $" },
+  { value: 30000, label: "30,000 $" },
+  { value: 25000, label: "25,000 $" },
+  { value: 20000, label: "20,000 $" },
+  { value: 15000, label: "15,000 $" },
+];
```

Comment:
That looks cool, no shit =)

But there is a better way:

```suggestion
const maxPrice = 200_000;
const minPrice = 15_000;
const priceStep = 5_000;

export const priceToTemplates = Array.from(
  { length: (maxPrice - (minPrice - priceStep)) / priceStep },
  (_, i) => {
    const value = i * 5000 + 15000;
    return { value, label: `${value.toLocaleString()} $` };
  }
);
```

And don't need to reverse this arra, reverse it where you are using it - in the computed